lib/routes/account.js

```diff
@@ -837,7 +838,11 @@ module.exports = (log, db, mailer, Password, config, customs, signinUtils, push)
 
         const res = {}
         return db.account(uid)
-          .then(account => {
+          .then(result => {
+            account = result
+            if (scope.contains('profile')) {
+              res.profileChangedAt = account.profileChangedAt
+            }
```

Comment:
This check seems redundant given the logic below where you check `Object.keys(res).length !== 0` and then add `profileChangedAt`.